Assets/Scripts/Model/Cards/Card.cs

```diff
@@ -23,6 +23,7 @@ public abstract class Card
         public bool Faceup { get; private set; } = false;
         public Information Information { get; private set; } = Information.HIDDEN_FROM_ALL;
         public bool Active { get; private set; } = false;
+        public abstract Stealable Stealable { get; }
```

Comment:
It's a great idea to put it in `Card`! It's a pattern similar to the `TrashOptions` below.